polynote-server/src/main/scala/polynote/server/repository/format/ipynb/IPythonFormat.scala

```diff
@@ -14,13 +14,13 @@ class IPythonFormat extends NotebookFormat {
 
   override val extension: String = "ipynb"
 
-  override def decodeNotebook(noExtPath: String, rawContent: String): RIO[BaseEnv with GlobalEnv, Notebook] = for {
+  override def decodeNotebook(noExtPath: String, rawContent: String): RIO[Any, Notebook] = for {
```

Comment:
Why was this change necessary?

Reply:
It was just to make it easier to directly use an `IPythonFormat` from tests (without having to pass an environment). No reason for this impl to ask for something it doesn't need.